aladdin/lib/aws/dns_mapping.py

```diff
@@ -2,11 +2,14 @@
 import logging
 from datetime import datetime
 
+from aladdin.lib.cluster_rules import ClusterRules
+
 
 def fill_hostedzone(
-    boto_session, cluster_domain_name, namespace_domain_name, hostnames_to_loadbalancers
+    boto_session, hostnames_to_loadbalancers, cluster_domain_name=None, namespace_domain_name=None
 ):
-    # Main DNS is on prod, sub DNS should be on sandbox
```

Comment:
was this comment misplaced (does it need to go somewhere else)?

Reply:
I think this is very specific to Fivestars and is not totally relevant for aladdin